framework/tlvf/AutoGenerated/src/tlvf/test/tlvVarList.cpp

```diff
@@ -266,6 +273,7 @@ bool tlvTestVarList::add_var1(std::shared_ptr<cInner> ptr) {
         TLVF_LOG(ERROR) << "Not enough available space on buffer";
         return false;
     }
+    m_var1_init = true;
```

Comment:
I gotta say I don't fully understand what is the difference between setting this boolean to true, to return true in the function.
If the function is failing, then this member will also be set to false.

Reply:
When you return true, the caller knows that add succeeded. But if the caller never called the add function, tlvf needs to be aware of that so in Finalize we check the value of this variable.